projects.md

```diff
@@ -0,0 +1,10 @@
+# Projects
+
+Existing projects in Go that can be used as examples.
+
+## HelloFresh
+
+- https://github.com/hellofresh/remote-config-service
```

Comment:
Maintaining a demo project is a burden that nobody is gonna carry on over time, although having a template/skeleton project is another goal and it's feasible.

So I like a separate repo for private project listing like @rafaeljesus suggests. If somebody creates such a repo in the org, I can move the links there.